src/Sylius/Bundle/AddressingBundle/Resources/config/validation/Country.xml

```diff
@@ -19,6 +19,10 @@
             <option name="groups">sylius</option>
         </constraint>
         <property name="code">
+            <constraint name="Country">
```

Comment:
Just thoughts out loud,

`Country` is available since sf2, but wasn't used, so I've a feeling there was some obscure reason for this, for example
-  due INTL country list not being complete or correct at the time.
- https://github.com/symfony/symfony/issues/36733

Reply:
Can't say I remember why `Regex` was used instead of `Country`. Might've been a lack of knowledge or it might not have been that well documented at the time.
From an operational standpoint, I don't see any issue with it, in the UI context we're getting the country choices from INTL either way, same happens with getting the country name from code and vice versa. This just ensures that the data we might get via API will be compatible with UI.
Alpha-3 doesn't apply here since there was always a `Length` constraint set to 2 (though I guess it could be also removed since `Country` takes care of it), so it has always been exclusively alpha-2.